### DISCUSSION TITLE
create multiple traces on same plot section for multiple x and y axis

### DISCUSSION THREAD

**Discussion by pankajAppPerfect** [Kanaries/pygwalker#379] [Q&A]
Is it possible to have multiple line traces on same plot with secondary axes.
For e.g. in below case i want both columns of y axes on same trace with 'A' on left y axis  and 'B' on right side axis.

![Screenshot 2024-01-03 at 11 40 47 AM](https://github.com/Kanaries/pygwalker/assets/117146644/e5344055-f05d-4838-9a31-d01f53d45825)

**Comment by brandonrwin** [comment]
Drag "Measure names" from the "Field List" on the left to "Color" that's in the middle column.
